static/app/views/releases/list/index.tsx

```diff
@@ -129,9 +135,48 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
        * uses shouldReload=true and there is no reloading happening.
        */
       forceCheck();
+      if (this.hasV2ReleaseUIEnabled) {
+        // Refetch new threshold statuses if  new releases are fetched
+        this.fetchThresholdStatuses();
+      }
     }
   }
 
+  fetchThresholdStatuses() {
+    const {selection, organization, api} = this.props;
+    const {releases} = this.state;
+
+    // Grab earliest release and latest release - then fetch all statuses within
+    let start = releases[0].dateCreated;
+    let end = releases[0].dateCreated;
+    const releaseVersions: string[] = [];
+    releases.forEach(release => {
+      if (release.dateCreated < start) {
+        start = release.dateCreated;
+      }
+      if (release.dateCreated > end) {
+        end = release.dateCreated;
+      }
+      releaseVersions.push(release.version);
+    });
+
+    const query: ThresholdStatusesQuery = {
+      start,
+      end,
+      release: releaseVersions,
+    };
+    if (selection.projects.length) {
+      query.project = this.getSelectedProjectSlugs();
```

Comment:
do we typically pass project slug or id as query params?

Reply:
I think usually for internal API's we use IDs, 
but since this will be a consumer facing api, slugs are more easily digestible - so i opted to utilize slug and version, rather than id's for project/release